frontend/src/App.js

```diff
@@ -1,15 +1,25 @@
 import React from 'react';
-import logo from './logo.svg';
+import { Route, Switch, BrowserRouter as Router } from "react-router-dom";
```

Comment:
There are some styling differences, but those can easily be handled soon with the adoption of an ESLint/Prettier config later.